src/img/icons/circle-close-error.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="17" height="17" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
height and width are not required tags in a svg file.

Also, why is the viewbox 17x17 ?